app/components/lists/fairs.js

```diff
@@ -1,193 +1,209 @@
-  //   paragraph2:
-  //     "A tradicional procissão faz-se em redor do santuário com a imagem de Nossa Senhora dos Remédios e no final há o tradicional canto das loas a Nossa Senhora, seguido da despedida dos Círios.",
-  //   paragraph3: "",
-  //   paragraph4: "",
-  //   sources:
-  //     "Retirado do site “https://jornaldascaldas.pt/2021/10/18/cirios-a-nossa-senhora-dos-remedios-2/”, consultado em setembro 23.",
-  //   isFeatured: true,
-  // },
-  // {
-  //   id: "fair-08",
-  //   title: "Feira de Castro",
-  //   month: new Date("2023-10-21"),
-  //   region: "Centro e Beira Interior",
-  //   district: "Beja",
-  //   town1: "Castro Verde",
-  //   town2: "Uniao das Freguesias de Castro Verde e Casével",
-  //   imgURL:
-  //     "https://barlavento.sapo.pt/wp-content/uploads/2021/08/Mina-de-Sal-gema-Loul%C3%A9-01.jpg",
-  //   description:
-  //     "Instituída por Filipe II, em 1620, em resposta a uma solicitação feita pelos moradores do concelho que pretendiam obter, com o rendimento dos terrádegos (imposto sobre o terreno ocupado pelas barracas e tendas dos feirantes), os fundos necessários à reconstrução da Igreja das Chagas do Salvador (ou Igreja de Nossa Senhora dos Remédios), a Feira de Castro depressa se tornou a principal feira do sul, primeiro de gado, depois de mil e uma mercadorias, manufaturas e produtos da terra, animando o comércio local e regional.",
-  //   paragraph1: "",
-  //   paragraph2:
-  //     "Durante três dias e encerrando, invariavelmente, numa tradição que se perpetua no terceiro domingo de outubro, que originou o dito popular de «tão certo como a Feira de Castro», a feira mantém vivos alguns elos com o passado, desde o tradicional arraial de gado à venda dos produtos do pequeno comércio e da indústria familiar, das mantas de lã aos queijos, dos frutos secos aos artefactos da latoaria, das quinquilharias às loiças de barro, das alfaias agrícolas ao mobiliário rústico.",
-  //   paragraph3:
-  //     "Com o passar dos tempos, a feira acabou por se moldar às exigências do presente e aos ditames da modernidade, tanto no comércio como na indústria do lazer e do divertimento. Porém, continua a mesma no apelo que faz ao convívio, à troca e ao encontro.",
-  //   paragraph4:
-  //     "Paralelamente à animação da própria feira, é dinamizado um conjunto de iniciativas que têm como referência o valorizar da tradição e do património cultural da região, merecendo destaque o desfile de corais Planície a Cantar ou o Encontro de Tocadores de Viola Campaniça e Cantadores de Despique e Baldão.",
-  //   sources:
-  //     "Retirado do Site “https://www.cm-castroverde.pt/pt/menu/943/feira-de-castro.aspx”, consultado em setembro 23.",
-  //   isFeatured: true,
-  // },
+  {
```

Comment:
why not mongo/firebase?

Reply:
Because this list is too big, I will spend a lot of time transferring the data to mongo. I want to do it later, I will add this to the issue's list but at this point I have other priorities